### ISSUE TITLE
Notes

### ISSUE THREAD

**Issue by JacksonBurns** [JacksonBurns/chemeleon#1]
I've tried the following:
 - naive autoencoders: vanishing gradient
 - regularized autoencoder: tied weights. Downstream performance not good
 - autoencoders with self-normalizing neural networks: overfit
 - regularized above with alpha dropout, tied weights still (i.e. denoising): downstream not good
 - tried overcomplete and undercomplete

at this point, changed fine-tuning approach to change the weights of the pretrained encoder as well

 - changed to masked training, no dropout, constant internal size: did OK downstream
 - changed decoder to single layer
